js/ui/tag_box.js

```diff
@@ -224,6 +225,15 @@ const TagBox = SelectBox.inherit({
         return this.callBase();
     },
 
+    _renderLabel: function() {
+        const options = {
+            editor: this,
+            container: this._$tagsContainer
+        };
+
+        renderLabel(options);
+    },
+
```

Comment:
```suggestion
    _getLabelOptions: function() {
        return {
            editor: this,
            container: this._$tagsContainer
        };
    },

```
After extraction we may override option getter only